pynestml/codegeneration/resources_nest/point_neuron/common/NeuronHeader.jinja2

```diff
@@ -566,6 +564,24 @@ private:
     MAX_SPIKE_RECEPTOR = {{ns.count}}
   };
 
+  enum ContinuousInput
+  {
+{%-       set ns.count = 0 -%}
+{%- for port in neuron.get_continuous_input_ports() %}
+{%-   if port.has_vector_parameter() -%}
+{%      set size = utils.get_numeric_vector_size(port) | int %}
+{%-     for i in range(size) %}
+    {{port.get_symbol_name().upper()}}_{{i}} = {{ns.count}},
+{%-       set ns.count = ns.count + 1 -%}
+{%-     endfor %}
+{%-   else %}
+    {{port.get_symbol_name().upper()}} = {{ns.count}},
+{%-     set ns.count = ns.count + 1 -%}
+{%-   endif -%}
+{%- endfor %}
+    MAX_CONTINUOUS_INPUT = {{ ns.count }}
```

Comment:
This name is a bit confusing and could correspond with a maximum continuous input value, rather than port index. Could you rename it into ``NUM_CONTINUOUS_INPUT_PORTS`` or ``MAX_CONTINUOUS_INPUT_PORT_INDEX`` or something similar?